### PR TITLE
KFSPTS-3599 Add new fields to Account Global document

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java [hunk 1]
```diff
@@ -168,4 +172,36 @@
...
+    public static final String ERROR_DOCUMENT_ACCT_GLB_MAINT_ICR_NOT_EMPTY_FOR_NON_CG_ACCOUNT = "error.document.accountGlobal.acct.cgICRNotEmptyForNonCGAccount";
+    public static final String ERROR_DOCUMENT_ACCT_GLB_MAINT_EXP_DT_AND_REMOVE_CONT_ACCT = "error.document.accountGlobal.acct.expirationDate.removeContAcct";
+
+    public static final String ERROR_DOCUMENT_ACCT_GLB_MAINT_REMOVE_INC_STR_CHART_AND_ACCT_CHECKED_WHEN_INC_STR_REQ = "error.document.accountGlobalMaintenance.removeIncStrChartAndAcctChecked.incStrReq";
+    public static final String ERROR_DOCUMENT_ACCT_GLB_MAINT_REMOVE_INC_STR_CHART_AND_ACCT_CHECKED_WHEN_INC_STR_REQ_FOR_ACCT = "error.document.accountGlobalMaintenance.removeIncStrChartAndAcctChecked.incStrReqForAcct";
+    public static final String ERROR_DOCUMENT_ACCT_GLB_MAINT_CLOSED_CHECKED_WHEN_ACCOUNT_HAS_OPEN_ENCUMBRENCES = "error.document.accountGlobalMaintenance.accountCannotCloseOpenEncumbrance";
+    public static final String ERROR_DOCUMENT_GLB_MAINT_CANNOT_BE_CLOSED_EXP_DATE_INVALID = "error.document.accountGlobalMaintenance.accountCannotBeClosedExpDateInvalid";
```
Comment (bhutchinson) [inline src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java]: This is nitpicky, but should it be ERROR_DOCUMENT_ACCT_GLB_MAINT...?

**Comment by bhutchinson** [comment]
I've looked through the updated changes and only had one question.

File: src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java (hunk 1 above)
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java]: hmmm, that is odd, I didn't add this stuff

**Comment by bhutchinson** [comment]
retest this please

**Comment by bhutchinson** [comment]
Since I only had one nitpicky thing and since Daniela is out, I'll merge this so it can move forward.

**Pull request merged**